### ISSUE TITLE
Up und Downvote mit Pfeiltasten

### ISSUE THREAD

**Issue by CookedApps** [Scarwolf/p0weruser#129]
## Allgemeine Informationen
**Browser:** Alle
**Version:** Alle
**Modul:** Widescreen Modus

## Beschreibung
Im Widescreen Modus kann man mit den Pfeiltasten Links/Rechts zum nächsten und vorherigen Post springen. Eine sinnvolle Erweiterung wäre mit den Hoch und Runter Pfeiltasten Voten zu können (Upvote/Downvote).

Falls es das schon gibt, zeigt mir gerne wo, habe es nicht gefunden. :thinking:

**Comment by CookedApps** [comment]
Nachtrag: Hoch/Runter Taste ist ggf. durch Bild hoch/runterscrollen besetzt, wenn man das Bild vergrößert hat.

**Comment by twobiers** [comment]
Eigentlich würde ich ungern Hotkeys vom nativen pr0gramm überschreiben. Habe beim Ausprobieren aber verstanden, wieso das im Widescreen Mode eine sinnvolle Ergänzung wäre. Problem an der Sache ist, dass es eben **nur** im Widescreen Mode sinnvoll ist. Weiß nicht, ob es klug ist, eigenes Verhalten zu integrieren. Außerdem sehe ich Probleme, sollte jemand die Pfeiltasten nutzen, um durch die Kommentare zu scrollen.

Zur Not kannst du dir recht easy ein eigenes Userscript basteln, der Code dafür dürfte ungefähr so aussehen:
```js
document.addEventListener('keydown', (event) => {
  event.preventDefault();
  switch(event.code) {
    case "ArrowUp":
      p.hotkeys.toggleVote(true);
      break;
    case "ArrowDown":
      p.hotkeys.toggleVote(false);
      break;
  }
});
```

**Comment by CookedApps** [comment]
Ja, da hast du recht. Außerdem sind die Pfeiltasten schon belegt im Widescreen Modus, zum Scrollen des angezeigten Bildes. Müsste man als konfigurierbar machen.
